pkg/agent/svid/rotator.go

```diff
@@ -65,19 +63,20 @@ func (r *rotator) Run(ctx context.Context) error {
 
 func (r *rotator) runRotation(ctx context.Context) error {
 	for {
+		if err := r.rotateSVID(ctx); err != nil {
```

Comment:
This change causes the rotator to evaluate the SVID for rotation right when Run is executed instead of waiting for the initial period (1.5s). The resolved a flaky unit-test and _shouldn't_ impact production behavior.